config/detekt/detekt.yml

```diff
@@ -25,6 +25,9 @@ coroutines:
 naming:
   FunctionNaming:
     ignoreAnnotated: ['Composable']
+  FunctionParameterNaming:
```

Comment:
We need this because of the:

```
@Part `data`: MultipartBody.Part,
```

The generator handles it as a [reserved word](https://openapi-generator.tech/docs/generators/kotlin/#reserved-words) even though it will work as a param name.